### PR TITLE
(feat) start.html buttons -and input field- are disabled after user clicks

### PR DESCRIPTION
**Pull request by i-made-that**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -27,3 +27,6 @@ dist/
 # see https://npmjs.org/doc/faq.html#Should-I-check-my-node_modules-folder-into-git
 node_modules/
 bower_components/
+
+# .DS_Store
+client/sender-app/content/.DS_Store
```
Comment (ajkitson) [inline .gitignore]: If you do **/*.DS_Store, it should ignore DS_Store files, wherever they are in the directory structure

File: client/sender-app/start/start.js [hunk 2]
```diff
@@ -31,6 +36,7 @@
     function startGame() {
       playerMessenger.ready();
       toastr.info(playerUser.getUser() + ' wants to play!');
+      vm.btnStart = true;
```
Comment (ajkitson) [inline client/sender-app/start/start.js]: Is vm.btnStart just a flag to indicate the player has clicked the ready button? If so, would be more intuitive to call it something like vm.ready or vm.playerReady. That way it's clear that other elements can reference the flag, not just the button

File: client/sender-app/start/start.js [hunk 3]
```diff
@@ -13,17 +13,22 @@
...
     changeStateListener();
 
     // user setting their name and revealing the start game button
     function setUser(event) {
+      var userName = vm.userName;
       // set username in playerUser factory
       playerUser.setUser(vm.userName);
       // unhide btn
       vm.btnShow = true;
+      vm.userName = '';
```
Comment (ajkitson) [inline client/sender-app/start/start.js]: I'm not clear on why username is moving around so much in here--why push it to a local variable and why clear it from vm?

File: .gitignore [hunk 4]
```diff
@@ -27,3 +27,5 @@ dist/
 # see https://npmjs.org/doc/faq.html#Should-I-check-my-node_modules-folder-into-git
 node_modules/
 bower_components/
+
+**/*.DS_Store
```
Comment (rmagee88) [inline .gitignore:31]: not sure we need to have this... think you can just delete it from your local repo... think it's just a weird OSX thing?

Comment (i-made-that) [inline .gitignore:31]: Definitely a weird, automatically-generated OSX thing.  I looked into it and I think we should actually keep it in the gitignore.  Check out the wikipedia wisdom:
".DS_Store files impose additional burden on revision control process: They are prone to changes and therefore appear in commits unless specifically excluded."
What you think?

**Pull request merged**